tests/views.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework_json_api.views import ModelViewSet, RelationshipView
+
+from .models import BasicModel
+
+
+class BasicModelViewSet(ModelViewSet):
```

Comment:
Same as above better to be moved to test_relations.py for now.

Reply:
This effectively depends on figuring out how to "inline" the urlconf, as I asked about in the other unresolved conversation in this review. Once we can figure out how to move the urlconf inline, this change should be pretty trivial.